rmw/include/rmw/rmw.h

```diff
@@ -98,11 +98,13 @@ extern "C"
 #include "rosidl_runtime_c/service_type_support_struct.h"
 #include "rosidl_runtime_c/sequence_bound.h"
 
+#include "rmw/event.h"
 #include "rmw/init.h"
+#include "rmw/event_callback_type.h"
```

Comment:
```suggestion
#include "rmw/event_callback_type.h"
#include "rmw/init.h"
```

nitpick, header files should be alphabetical order?